CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 
 ### Changed
-- Updated minimum Bitmovin Player version to 3.41.0 and raised minimum deployment targets to iOS/tvOS 14.0
+- Updated minimum Bitmovin Player version to 3.62.0 and raised minimum deployment targets to iOS/tvOS 14.0
```

Comment:
I understand that this entry already existed but actually you did not bump the minimum Player version to `3.62.0` 🤔 (it also is not set to `3.41.0`)

There is no need for changes; we will handle this internally.